src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -387,7 +387,18 @@ public virtual bool IsTupleField
         /// <summary>
         /// Returns True when field symbol is not mapped directly to a field in the underlying tuple struct.
         /// </summary>
-        internal virtual bool IsVirtualTupleField
+        public virtual bool IsVirtualTupleField
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        /// <summary>
+        /// Returns true if this is a field representing a Default element like Item1, Item2...
+        /// </summary>
+        public virtual bool IsDefaultTupleElement
```

Comment:
I think I will keep the names as-is. They are internal APIs for now anyways. If we make them public, we will have another review, and possibly change the names anyways

---
In reply to: [78256421](https://github.com/dotnet/roslyn/pull/13713#discussion_r78256421) [](ancestors = 78256421,78254531)